### DISCUSSION TITLE
Update Name of People

### DISCUSSION THREAD

**Discussion by loelismurf** [liedekef/events-made-easy#457] [Q&A] [answered]
If i change de User name and prename, so it not get updatet in the Database of EME, how can i update it or is it a BUG?

**Comment by liedekef** [comment]
I'm not sure I understand your question. If you change the name of a WP user *and* that user is linked to a person in EME, then that EME person should get updated too.

**Comment by loelismurf** [comment]
Here WP username
![image](https://github.com/liedekef/events-made-easy/assets/77551562/daaabd31-d837-45d1-8083-d085fe88f853)

after Update on Dropdownfields are not Updated
![image](https://github.com/liedekef/events-made-easy/assets/77551562/2649af64-7963-4884-bd03-2ac385105867)

In Backend on EME "person" it is updatet
![image](https://github.com/liedekef/events-made-easy/assets/77551562/d829025c-242e-4573-8f86-c553bc66456f)

**Reply by liedekef** [reply, marked as answer]
In the author and contact, I show the "display name" of a person.

**Reply by loelismurf** [reply]
omg my fault i have change the Name but Worpress do not change public name "display name" so my Fault xD
Thanks für answer
